# 02 — Baseline

LogisticRegression на сырых фичах + Dummy.

In [ ]:
import pandas as pd

from src.modeling import train_and_eval_baseline


In [2]:
reports = train_and_eval_baseline()
table = pd.DataFrame({k: v.to_dict() for k, v in reports.items()}).T
table


,roc_auc,pr_auc,f1,precision,recall,accuracy,tn,fp,fn,tp
dummy_stratified,0.500399,0.221396,0.221045,0.221884,0.220211,0.656599,3643.0,1024.0,1034.0,292.0
logreg_baseline,0.762882,0.536221,0.528541,0.496032,0.565611,0.776740,3905.0,762.0,576.0,750.0


## Комментарий

Baseline LogReg c `class_weight='balanced'` даёт `ROC-AUC ≈ 0.76` vs `0.5` у stratified dummy, что подтверждает разделимость классов. В CP2 расширим модели (RF/XGBoost/LightGBM), добавим FE и тюнинг гиперпараметров.